### DISCUSSION TITLE
Installing PowerDNS-Admin to manage existing PowerDNS installation

### DISCUSSION THREAD

**Discussion by mbh-dev** [ngoduykhanh/PowerDNS-Admin#1704] [Q&A]
### PDA version

0.4.1

### Python version

3.0

### Steps to Reproduce

1. Existing server setup and installation of PowerDNS
2. New docker installation of PowerDNS-Admin
3. Error 500 when trying to create new zones through PowerDNS-Admin 

### Expected Behavior

New zone should be created

### Observed Behavior

Currently when I try to create a new zone I receive an error 500 message, I also have the API setup but it's not reading the existing zones.

Do I need do a manual docker-compose.yml to point to the existing PowerDNS server database or would this cause issues?

Thanks

**Comment by vdchuyen** [comment]
You can debug with just PowerDNS API first (or using pdnsutil check-zones) , this is just a wrapper around that API, whenever that's API works, this also works.

**Comment by AzorianMatt** [comment]
@mbh-dev This was inappropriate to report this as a bug. This is an issue with your deployment that needs debugged. I'm converting this to a discussion.
